### PR TITLE
add QuCmp

### PR DESCRIPTION
**Pull request by Roger-luo**

### CODE REVIEW

**Comment by i2000s** [comment]
@Roger-luo As we have committed [the renaming](https://github.com/JuliaQuantum/QuCmp.jl/issues/7), could you update the link? I am sure you are preparing a detailed blog post. Looking forward to it!

File: projects/index.md [hunk 1]
```diff
@@ -57,3 +57,13 @@ Schrodinger Equation, Liouville von Neumann Equation, Lindblad Master Equation.
 have been integrated from packages like [ODE.jl](https://github.com/JuliaLang/ODE.jl), [ExpmV.jl](https://github.com/marcusps/ExpmV.jl),
 [Expokit.jl](https://github.com/acroy/Expokit.jl). Currently it depends on and makes use of features from
 [QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl).
+
+
+### [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp)
+
+A framework for simulations of quantum computing and quantum information processing.
```
Comment (i2000s) [inline projects/index.md]: for simulating quantum information processing.

File: projects/index.md [hunk 2]
```diff
@@ -57,3 +57,13 @@ Schrodinger Equation, Liouville von Neumann Equation, Lindblad Master Equation.
...
+As quantum computation implementation in experiments grows rapidly, a toolkit for bridging the gap between experimentalist and theorists could be necessary. To be specific, Translating quantum algorithm to quantum circuits or Hamiltonians in other models with a lower resource cost is important for both experimentalists and theorists.
```
Comment (i2000s) [inline projects/index.md]: You may not need this sentence at all.

File: projects/index.md [hunk 3]
```diff
@@ -57,3 +57,12 @@ Schrodinger Equation, Liouville von Neumann Equation, Lindblad Master Equation.
 have been integrated from packages like [ODE.jl](https://github.com/JuliaLang/ODE.jl), [ExpmV.jl](https://github.com/marcusps/ExpmV.jl),
 [Expokit.jl](https://github.com/acroy/Expokit.jl). Currently it depends on and makes use of features from
 [QuBase.jl](https://github.com/JuliaQuantum/QuBase.jl).
+
+
+### [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp)
```
Comment (i2000s) [inline projects/index.md]: Link to https://github.com/JuliaQuantum/QuCmp.jl
Thanks.

**Comment by i2000s** [comment]
@Roger-luo When you finish, could you squash the commits? I will be helping on side and merge afterward. Thank you.

**Comment by Roger-luo** [comment]
squashed @i2000s

**Comment by i2000s** [comment]
It would be nice to write something on the project and call for attentions. I will merge this first.

**Pull request merged**